--- php.ini-development.orig
+++ php.ini-development
@@ -481,7 +481,9 @@
 ; Development Value: E_ALL
 ; Production Value: E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; http://php.net/error-reporting
-error_reporting = E_ALL
+;
+; E_ALL | E_STRICT (Catch all error and warnings, including coding standard warnings.)
+error_reporting = E_ALL | E_STRICT
 
 ; This directive controls whether or not and where PHP will output errors,
 ; notices and warnings too. Error output is very useful during development, but
@@ -763,7 +765,7 @@
 
 ; Directory in which the loadable extensions (modules) reside.
 ; http://php.net/extension-dir
-;extension_dir = "./"
+extension_dir = "/usr/lib/php/modules/"
 ; On windows:
 ;extension_dir = "ext"
 
@@ -918,30 +920,30 @@
 ;   Be sure to appropriately set the extension_dir directive.
 ;
 ;extension=bz2
-;extension=curl
+;extension=calendar
+extension=curl
+;extension=dba
+;extension=enchant
+;extension=exif
 ;extension=ffi
 ;extension=ftp
-;extension=fileinfo
 ;extension=gd
 ;extension=gettext
 ;extension=gmp
-;extension=intl
+;extension=iconv
 ;extension=imap
+;extension=intl
 ;extension=ldap
-;extension=mbstring
-;extension=exif      ; Must be after mbstring as it depends on it
 ;extension=mysqli
-;extension=oci8_12c  ; Use with Oracle Database 12c Instant Client
-;extension=oci8_19  ; Use with Oracle Database 19 Instant Client
 ;extension=odbc
-;extension=openssl
-;extension=pdo_firebird
+zend_extension=opcache
+;extension=pdo_dblib
 ;extension=pdo_mysql
-;extension=pdo_oci
 ;extension=pdo_odbc
 ;extension=pdo_pgsql
 ;extension=pdo_sqlite
 ;extension=pgsql
+;extension=pspell
 ;extension=shmop
 
 ; The MIBS data available in the PHP distribution must be installed.
@@ -952,10 +954,13 @@
 ;extension=sockets
 ;extension=sodium
 ;extension=sqlite3
+;extension=sysvmsg
+;extension=sysvsem
+;extension=sysvshm
 ;extension=tidy
 ;extension=xsl
 
-;zend_extension=opcache
+extension=zip
 
 ;;;;;;;;;;;;;;;;;;;
 ; Module Settings ;
@@ -1758,10 +1763,10 @@
 
 [opcache]
 ; Determines if Zend OPCache is enabled
-;opcache.enable=1
+opcache.enable=1
 
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
-;opcache.enable_cli=0
+opcache.enable_cli=1
 
 ; The OPcache shared memory storage size.
 ;opcache.memory_consumption=128
@@ -1846,7 +1851,8 @@
 
 ; Protect the shared memory from unexpected writing during script execution.
 ; Useful for internal debugging only.
-;opcache.protect_memory=0
+; catch invalid updates of shared memory
+opcache.protect_memory=1
 
 ; Allows calling OPcache API functions only from PHP scripts which path is
 ; started from specified string. The default "" means no restriction
